cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -39,7 +39,13 @@ using arrow::int32;
 
 class TestProjector : public ::testing::Test {
  public:
-  void SetUp() { pool_ = arrow::default_memory_pool(); }
+  void SetUp() {
+    pool_ = arrow::default_memory_pool();
+    // Setup arrow log severity threshold to debug level.
+    arrow::util::ArrowLog::StartArrowLog("", arrow::util::ArrowLogLevel::ARROW_DEBUG);
+    // To test the eviction, uncomment the line below:
+    //setenv("GANDIVA_CACHE_SIZE", "10240", 1);
```

Comment:
Remove these lines if they are not necessary

Reply:
Removed, it is not necessary for projector itself, I just used to check some behaviors on this test suite case.